src/main/java/core/basesyntax/services/FruitShopService.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax.services;
+
+import core.basesyntax.services.operations.Operable;
+import java.util.List;
+import java.util.Map;
+
+public class FruitShopService {
+    public boolean processFile(String inputFilePath, String outputFilePath,
+                               Map<String, Operable> operations) {
+        DataFileReader fileReader = new DataFileReader();
+        DataFileWriter fileWriter = new DataFileWriter();
+        DataToMapParser parser = new DataToMapParser();
```

Comment:
Думаю что всякие ридеры и прочее, лучше передавать в класс, а не жестко привязывать инициализацию к нему. Чтоб в дальнейшем была возможность гибкого выбора инструментов для того же считывания или записи.

Reply:
\+

Reply:
I created this class only for simplifying ```MainApp```, instead of putting all actions there, so that ```MainApp``` only starts actions and prints message about result. But I've added constructor with this parameters